ui/app/templates/components/auth-jwt.hbs

```diff
@@ -46,22 +46,25 @@
   <div data-test-yield-content>
     {{yield}}
   </div>
-  <button
-    data-test-auth-submit={{true}}
-    type="submit"
-    disabled={{@disabled}}
-    class="button is-primary {{if @disabled 'is-loading'}}"
-    id="auth-submit"
-  >
-    {{#if this.isOIDC}}
-      {{#if this.role.providerButtonComponent}}
-        {{component this.role.providerButtonComponent}}
```

Comment:
Before we were rendering some custom divs within the button, now we just pass the provider icon if exists and OIDC

Reply:
Auth0 example compare
<img width="1440" alt="Screenshot 2023-10-17 at 1 26 17 PM" src="https://github.com/hashicorp/vault/assets/82459713/992d57ce-52c3-4f87-b481-49813f50c883">